core-cart.mjs

```diff
@@ -2,9 +2,90 @@ import {carts, products, promoCode, users} from "./dataset.mjs";
 
 import * as fs from "fs";
 import * as os from "os";
+import { moveMessagePortToContext } from "worker_threads";
 
-const discountedPrice = (price, rate = 0.10) => (price * (1 - rate)).toFixed(2);
+//INTESTAZIONE SCONTRINO
+const heading = function(){
+    let nMacchina='NOMEMACCHINA Cart - 43874\n'
```

Comment:
il nome macchina l'abbiamo visto in classe e il codice è pubblicato qui: https://github.com/danielebarbaro/node-cart-receipt-exercise/blob/partial-solution-tests/core-cart.mjs#L10